src/main/resources/migration/V0_4__Create_question_table.sql

```diff
@@ -0,0 +1,8 @@
+CREATE DATABASE question(
+    id varchar(250) primary key,
+    order int,
+    content text,
+    required bool,
+    id varchar(250) REFERENCES form(id),
+    id varchar(250) REFERENCES question_type(id)
```

Comment:
for the FK, 
form_id referecence....
question_type_id ....

And id lenth should be 255 for all